### PR TITLE
Avoid changing roots in grub if running abroot on previous partition

### LINKED ISSUE #180: running abroot on the previous volume
If you run abroot on the previous volume instead of the current one it will install to the current partition but still select the previous partition as the current one. 

To the user this would look like nothing changed apart from the default entry in grub. 

Maybe it would make sense to require a rollback before.

### PR DESCRIPTION
**Pull request by matbme**
closes #180

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: core/root.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ type ABRootManager struct {
 
 // ABRootPartition represents an ABRoot partition
 type ABRootPartition struct {
-	Label        string // a,b
+	Label        string // vos-a, vos-b
```
Comment (taukakao) [inline core/root.go:30]: ABRoot should be Distro Agnostic.

Comment (matbme) [reply core/root.go:30]: This is more of an example. Label gets whatever is in `partLabelA` and `partLabelB` inside `abroot.json`

Comment (matbme) [reply core/root.go:30]: I'll try to be more explicit about that

Comment (mirkobrombin) [reply core/root.go:30]: This is an example

**Review by mirkobrombin — APPROVED** [review]

**Pull request merged**